cds/modules/deposit/static/js/cds_deposit/avc/components/cdsUploader.js

```diff
@@ -208,14 +208,18 @@ function cdsUploaderCtrl(
         'Content-Type': (file.type || '').indexOf('/') > -1 ? file.type : ''
       },
       file.headers || {}
-    );
+    ),
+      url = urlBuilder.bucketVideo({
```

Comment:
@drjova can you please double check this?
I cannot easily use the `guessEndpoint` because now it is stateless, so I directly copied here what I needed, but I am not sure that this will work in all the cases.